File: sample/sample/AdOverlayViewController.m
//
//  AdOverlayViewController.m
//  sample
//
//  Created by Vladas Drejeris on 04/05/15.
//  Copyright (c) 2015 Adform. All rights reserved.
//

#import "AdOverlayViewController.h"

NSInteger const kMasterTag = 142636;

@interface AdOverlayViewController ()

@end

@implementation AdOverlayViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    // Initialize AFAdOverlay object with master tag.
    self.adOverlay = [[AFAdOverlay alloc] initWithMasterTagID:kMasterTag];
    
    // Display the ad at least 2 seconds after pushing the view controller.
    [self.adOverlay showFromViewController:self];
}

@end

File: sample/sample/Placements/AdOverlayViewController.swift
//
//  AdOverlayViewController.swift
//  sample
//
//  Created by Laurynas Letkauskas on 2021-02-26.
//

import AdformAdvertising
import UIKit

fileprivate let kMasterTag = 987063

class AdOverlayViewController: UIViewController {
    // AFAdOverlay placement requires strong reference to it, otherwise the object will be released by arc before loading and displaying the ad.
    var adOverlay: AFAdOverlay?

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.

        // Initialize AFAdOverlay object with master tag.
        adOverlay = AFAdOverlay(masterTagID: kMasterTag)

        // Display the ad at least 2 seconds after pushing the view controller.
        adOverlay?.show(from: self)
    }
}
